index.bs

```diff
@@ -5544,7 +5545,8 @@ enum DirectionSetting { "" /* horizontal */, "rl", "lr" };
 enum LineAlignSetting { "start", "center", "end" };
 enum PositionAlignSetting { "line-left", "center", "line-right", "auto" };
 enum AlignSetting { "start", "center", "end", "left", "right" };
-[Exposed=Window]
+[Exposed=Window,
+ Constructor(double startTime, unrestricted double endTime, DOMString text)]
```

Comment:
Constructor an extended attribute has been removed from Web IDL. Instead of this, you'll need to add "unrestricted" below, I'll make a suggestion.

Reply:
And this change here is no longer needed. Just the constructor operation below.

Reply:
Changed back to `[Exposed=Window]` as requested